src/napari_imagej/_function.py

```diff
@@ -149,25 +178,41 @@ def _preprocess_remaining_inputs(
     """Resolves each input in unresolved_inputs"""
     resolved_java_args = _convert_inputs(user_resolved_inputs)
     # resolve remaining inputs
-    for i in range(len(unresolved_inputs)):
-        name = unresolved_inputs[i].getName()
-        obj = resolved_java_args[i]
-        if obj is None:
+    for module_item, input in zip(unresolved_inputs, resolved_java_args):
```

Comment:
It is, in fact, done in another PR. To prevent a nightmare of a rebase, I'm just going to do the testing in that PR. See [this reminder](https://github.com/imagej/napari-imagej/pull/11#issuecomment-1119806382) I just wrote.